diff --git a/.gitignore b/.gitignore
index 9e59c9c4..5f6e6956 100644
--- ./.gitignore
+++ ./.gitignore
@@ -58,3 +58,4 @@ aclocal.m4
 /auto-build-save
 .deps
 /*.exe
+var/
diff --git a/md-convert b/md-convert
index a48689a7..46543c38 100755
--- ./md-convert
+++ ./md-convert
@@ -204,8 +204,6 @@ def find_man_substitutions():
     mtime = 0
 
     git_dir = srcdir + '.git'
-    if os.path.lexists(git_dir):
-        mtime = int(subprocess.check_output(['git', '--git-dir', git_dir, 'log', '-1', '--format=%at']))
 
     # Allow "prefix" to be overridden via the environment:
     env_subs['prefix'] = os.environ.get('RSYNC_OVERRIDE_PREFIX', None)
