diff --git a/.gitignore b/.gitignore
--- ./.gitignore
+++ ./.gitignore
@@ -58,3 +58,4 @@ aclocal.m4
 /auto-build-save
 .deps
 /*.exe
+var/
diff --git a/configure.ac b/configure.ac
--- ./configure.ac
+++ ./configure.ac
@@ -529,21 +529,6 @@ AC_ARG_ENABLE([zstd],
         AS_HELP_STRING([--disable-zstd], [disable to omit zstd compression]))
 AH_TEMPLATE([SUPPORT_ZSTD],
 [Undefine if you do not want zstd compression.  By default this is defined.])
-if test x"$enable_zstd" != x"no"; then
-    if test x"$ac_cv_header_zstd_h" = x"yes"; then
-	AC_MSG_RESULT(yes)
-	AC_SEARCH_LIBS(ZSTD_minCLevel, zstd,
-	    [AC_DEFINE(SUPPORT_ZSTD)],
-	    [err_msg="$err_msg$nl- Failed to find ZSTD_minCLevel function in zstd lib.";
-	     no_lib="$no_lib zstd"])
-    else
-	AC_MSG_RESULT(no)
-	err_msg="$err_msg$nl- Failed to find zstd.h for zstd compression support.";
-	no_lib="$no_lib zstd"
-    fi
-else
-    AC_MSG_RESULT(no)
-fi
 
 AC_MSG_CHECKING([whether to enable LZ4 compression])
 AC_ARG_ENABLE([lz4],
diff --git a/md-convert b/md-convert
--- ./md-convert
+++ ./md-convert
@@ -204,8 +204,6 @@ def find_man_substitutions():
     mtime = 0
 
     git_dir = srcdir + '.git'
-    if os.path.lexists(git_dir):
-        mtime = int(subprocess.check_output(['git', '--git-dir', git_dir, 'log', '-1', '--format=%at']))
 
     # Allow "prefix" to be overridden via the environment:
     env_subs['prefix'] = os.environ.get('RSYNC_OVERRIDE_PREFIX', None)
